doc/template.html

```diff
@@ -88,6 +88,11 @@ <h1>Node.js __VERSION__ documentation</h1>
       } else if (userSettings === 'true') {
         document.body.classList.add('dark-mode');
       }
+      if (document.body.classList.includes("dark-mode")) {
+        document.documentElement.setAttribute("data-color-scheme", "dark");
+      } else {
+        document.documentElement.setAttribute("data-color-scheme", "light");
+      }
```

Comment:
```suggestion
      if (document.body.classList.includes('dark-mode')) {
        document.documentElement.setAttribute('data-color-scheme', 'dark');
      } else {
        document.documentElement.setAttribute('data-color-scheme', 'light');
      }
```
Use single quotation marks for consistency.